SystemInterface/Diagnostics/IProcess.cs

```diff
@@ -73,6 +78,27 @@ public interface IProcess
         [MonitoringDescription("ProcessStandardOutput"), DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden), Browsable(false)]
         IStreamReader StandardOutput { get; }
 
+        /// <summary>
+        ///  Gets a new System.Diagnostics.Process component and associates it with the currently active process.
+        /// </summary>
+        /// <returns>A new System.Diagnostics.Process component associated with the process resource that is running the calling application.</returns>
+        Process GetCurrentProcess();
```

Comment:
The point of this library is test-ability of the code and easy mocking. If the return value cannot be mocked, there is no point in implementing this API in this library.